packages/backend-common/src/reading/GitlabUrlReader.ts

```diff
@@ -256,4 +256,69 @@ export class GitlabUrlReader implements UrlReader {
     const { host, token } = this.integration.config;
     return `gitlab{host=${host},authed=${Boolean(token)}}`;
   }
+
+  async getGitlabFetchUrl(target: string): Promise<string> {
+    // If the target is a raw API url then trust that no parsing is needed
```

Comment:
Hm should this be made more strict? As in, should we check that it at least refers to something blob-ish rather than ANY api route?

Reply:
I was rereading the Backstage documentation yesterday and noticed in [the section on the `read` API for URL readers](https://backstage.io/docs/plugins/url-reader#read) it claims

> read method expects a user-friendly URL, something which can be copied from the browser naturally when a person is browsing the provider in their browser.

which this bit is a contradiction of, so I'm happy to remove it.